gprofiler/utils/__init__.py

```diff
@@ -429,6 +461,10 @@ def atomically_symlink(target: str, link_node: str) -> None:
     """
     tmp_path = link_node + ".tmp"
     os.symlink(target, tmp_path)
+    # Added to fix bug where gprofiler crashes if link_node already exists.
```

Comment:
On Windows, you cannot `os.rename` a file if the target already exists?

Reply:
Yes, you cannot. You'll get something like the following error, if you don't first delete the symbolic link.

This will only work one time because the symbolic link doesn't exist.

tmp_path = "tma-shortcut.tmp"
os.symlink("some-file", tmp_path)
os.rename(tmp_path, "tma-shortcut.txt")

If that codepath is called again, you'll get an error: 
FileExistsError: [WinError 183] Cannot create a file when that file already exists: 'tma-shortcut.tmp' -> 'tma-shortcut.txt'

Reply:
I see. The idea with the `rename` is to have it atomic - i.e at any given point, `last_profile` is ALWAYS a link to the last profile. So let's put this part under `if is_windows() and os.path.exists(link_node)`, and also add a comment explaining that Windows cannot override a link with `rename`.

Reply:
Sounds good! Done